unit/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -714,3 +714,44 @@ SCENARIO(
     }
   }
 }
+
+SCENARIO(
+  "Logical Right-shift expressions are converted to SMT terms",
+  "[core][smt2_incremental]")
+{
+  GIVEN("An integer bitvector")
+  {
+    const auto one_bvint = from_integer({1}, signedbv_typet{8});
+
+    WHEN("We construct a lshr_exprt and convert it to an SMT term")
+    {
+      const auto shift_expr = lshr_exprt{one_bvint, one_bvint};
+      const auto constructed_term = convert_expr_to_smt(shift_expr);
+
+      THEN("We should get an logical shift left SMT term")
+      {
+        const smt_termt smt_term_one = smt_bit_vector_constant_termt{1, 8};
+        const auto expected_term = smt_bit_vector_theoryt::logical_shift_right(
+          /* Value to be shifted */
```

Comment:
Make the code say what it is doing not the comment! For example if you rename this instance of `smt_term_one` to `to_be_shifted` then the comment would be a duplicate which can be removed and the end result is more readable than the commented code is. Of course that means you need two different term variables then for the two operands, but I asked for that in the preceding comment already.